examples/ts-without-babel/webpack.config.js

```diff
@@ -0,0 +1,40 @@
+const ReactRefreshPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+const isDevelopment = process.env.NODE_ENV !== 'production';
+
+module.exports = {
+  mode: isDevelopment ? 'development' : 'production',
+  entry: './src/index.tsx',
+
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        include: path.join(__dirname, 'src'),
+        use: isDevelopment ? [
+          {
+            loader: 'babel-loader',
+            options: { plugins: ['react-refresh/babel'] }
+          },
+          'awesome-typescript-loader'
+        ]
+        : 'awesome-typescript-loader'
+      },
+    ]
```

Comment:
```suggestion
        use: [
          isDevelopment && {
            loader: 'babel-loader',
            options: { plugins: ['react-refresh/babel'] }
          },
          'awesome-typescript-loader'
        ].filter(Boolean)
      },
    ]
```

Reply:
Yes, it's better for consistency - applied. But FYI personally I don't like this hacky style. It's webpack 4. Initially I used `module.exports = (env, argv) => { ... }` and `if (argv.mode === 'development')`. In this case NODE_ENV and cross-env are not needed.

Reply:
I again think this is down to personal preference - using functions would be fine for me too, but just wanted to keep consistency between examples.